Orchestrator/docs/BFOrchestratorReport.md

```diff
@@ -1,109 +1,146 @@
+This section reports on utterances with duplicate or multiple labels. A duplicate utterance is detected when it is present more than once. Thus, the report lists the utterances tagged with more than one labels. Sometimes some dataset might contain utterances tagged with the same labels multiple times.
+
 The report also lists the redundancy.
 
 - Multi-label utterances and their labels
 - Duplicate utterance and label pairs
 
+### How to use this section
+
+**TBD**
+
+
+
 ## Ambiguous
 
-For an evaluation utterance, if an Orchestrator model correctly predicts its intent label, then it's
-a true positive prediction. However every prediction comes with a score, which is
-essentially the probability and confidence for the prediction.
-If the Orchestrator model also makes a high-score prediction close to that of the correctly predicted
-label, then we call such a prediction "ambiguous."
+This section reports on utterances ambiguous predictions. For an evaluation utterance, if an Orchestrator model correctly predicts its intent label, then it's a true positive prediction. However every prediction comes with a score, which is essentially the probability and confidence for the prediction. If the Orchestrator model also makes a high-score prediction close to that of the correctly predicted label, then we call such a prediction "ambiguous."
 
 In this section, the report lists all the utterances with an ambiguous prediction in a table.
 The table has several columns:
 
-- Utterance         --
+- Utterance         -- the utterance
 - Labels            -- the true labels for the utterance
 - Predictions       -- the labels predicted by the Orchestrator model
 - Close Predictions -- some other labels predicted with a close high score to that of the predicted label. 
 
 Besides the prediction score, the report also shows the closest example to the utterance
 within the label's utterance set.
 
+### How to use this section
+
+**TBD**
+
+
+
+Sometimes some dataset might contain utterances tagged with the same labels multiple times.
+The report also lists this redundancy.
+
+- Multi-label utterances and their labels
+- Duplicate utterance and label pairs
+
+### How to use this section
+
+**TBD**
+
+### 
+
 ## Misclassified
 
-For an evaluation utterance, if an Orchestrator model falsely predicts its intent label, then
-this prediction is a mis-classified case.
-Usually the label wiht the highest prediction score is chosen as the predicted label, but
+This section reports on utterances with incorrect predictions. An a misclassified predication is one in which an Orchestrator model falsely predicts its intent label. Usually the label with the highest prediction score is chosen as the predicted label, but
 it can be different from the ground-truth label for the utterance.
 
 Similar to the last section, the report also lists the prediction and ground-truth labels with
 their prediction scores and closest examples.
 
+
+
+### How to use this section
+

```

Comment:
Just remove the TBD?  Can always add whatever we need to add later?

Reply:
Hung-chih was planning to work on it once I'm done here. This is his placeholder.